### DISCUSSION TITLE
为什么Ubuntu使用v2raya会被ChatGPT判定为代理

### DISCUSSION THREAD

**Discussion by VeddySama** [v2rayA/v2rayA#1462] [Q&A] [answered]
是因为代理规则没配置对吗，一登录就显示关闭VPN

**Comment by Wakotu** [comment, marked as answer]
用透明代理模式会被看出来，你浏览器配置20171或者20172的那个http端口作为代理就可以了

**Reply by happyTonakai** [reply]
感谢，在系统设置中-Network-Network Proxy中配置一下就可以了

**Reply by Aike2071** [reply]
> 感谢，在系统设置中-Network-Network Proxy中配置一下就可以了

请问可以说一下怎么填Network Proxy中的配置吗？

**Reply by happyTonakai** [reply]
> > 感谢，在系统设置中-Network-Network Proxy中配置一下就可以了
> 
> 请问可以说一下怎么填Network Proxy中的配置吗？

HTTP/HTTPS Proxy: localhost 20172
Socks Host: localhost 20170

**Reply by Aike2071** [reply]
> > > 感谢，在系统设置中-Network-Network Proxy中配置一下就可以了
> > 
> > 
> > 请问可以说一下怎么填Network Proxy中的配置吗？
> 
> HTTP/HTTPS Proxy: localhost 20172 Socks Host: localhost 20170

非常感谢！
